packages/console/src/Console.tsx

```diff
@@ -251,6 +253,10 @@ export class Console extends PureComponent<ConsoleProps, ConsoleState> {
     if (props.objectMap !== prevProps.objectMap) {
       this.updateObjectMap();
     }
+
+    if (state.isStuckToBottom) {
+      this.scrollConsoleHistoryToBottom();
```

Comment:
I don't know if it's an optimization that matters, but we could also check that it's not already scrolled to the bottom before scrolling here.

Not sure if telling an element to scroll to the position it's already at is expensive in any way in the DOM.

Reply:
I tried looking into this, don't feel like this adds much value, given that the ways I was trying seemed quite inefficient. Unless this is really needed, I think the way we have it currently is simpler to understand + does not require unnecessary DOM manipulations / operations.

Reply:
If it does pose a performance issue we notice in the future it should be easy enough to fix.

You would just take the `scrollHeight - clientHeight >= 1` or w/e the check is and put it here as well so it's `isStuckToBottom && notAtBottom`

Reply:
Fair enough, will retest with change and repush